.github/workflows/main.yml

```diff
@@ -29,107 +30,85 @@ jobs:
+          working-directory: litmus-e2e/CypressE2E/
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
+
+      - name: Installing Cypress and running tests for Workflow page
+        if: startsWith(github.event.comment.body, '/run-e2e-workflow')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/Workflow/*
+          working-directory: litmus-e2e/CypressE2E/
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
+
+      - name: Install Cypress and run all tests
+        if: startsWith(github.event.comment.body, '/run-e2e-all')
+        uses: cypress-io/github-action@v2
+        continue-on-error: false
+        with:
+          wait-on: http://localhost:3000
+          spec: cypress/integration/**/*
+          working-directory: litmus-e2e/CypressE2E
+#           record: true
+#         env:
+#           # pass the Dashboard record key as an environment variable
+#           CYPRESS_RECORD_KEY: ${{ secrets.RECORDING_KEY }}
 
-      - name: Running node-memory-hog chaos experiment
-        if: startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') || startsWith(github.event.comment.body, '/run-e2e-all')
-        uses: mayadata-io/github-chaos-actions@v0.1.1
-        env:
-          INSTALL_LITMUS: true
-          EXPERIMENT_NAME: node-memory-hog
-          EXPERIMENT_IMAGE: litmuschaos/ansible-runner
-          EXPERIMENT_IMAGE_TAG: ci
-          IMAGE_PULL_POLICY: IfNotPresent
-          LITMUS_CLEANUP: true      
-                      
       - name: Check the test run
         if: |
-         startsWith(github.event.comment.body, '/run-e2e-pod-delete') || startsWith(github.event.comment.body, '/run-e2e-container-kill') ||
-         startsWith(github.event.comment.body, '/run-e2e-node-cpu-hog') || startsWith(github.event.comment.body, '/run-e2e-node-memory-hog') ||
-         startsWith(github.event.comment.body, '/run-e2e-all')
+         startsWith(github.event.comment.body, '/run-e2e-login') || startsWith(github.event.comment.body, '/run-e2e-workflow') || startsWith(github.event.comment.body, '/run-e2e-all')
         run: |
           echo ::set-env name=TEST_RUN::true
-
+  
       - name: Check for all the jobs are succeeded
         if: ${{ success() && env.TEST_RUN == 'true' }}
         uses: peter-evans/create-or-update-comment@v1
         with:
           comment-id: "${{ github.event.comment.id }}"
-          body: |  
+          body: |
             **Test Result:** All tests are passed
-          reactions: hooray         
+            **Run ID:** [${{ env.RUN_ID }}](https://github.com/litmuschaos/chaos-operator/actions/runs/${{ env.RUN_ID }})
```

Comment:
@Jonsy13 please update this 
it should be `**Run ID:** [${{ env.RUN_ID }}](https://github.com/litmuschaos/litmus/actions/runs/${{ env.RUN_ID }})`

Reply:
done.